website/.spelling

```diff
@@ -778,6 +778,10 @@ APPROX_COUNT_DISTINCT_DS_THETA
 APPROX_QUANTILE_DS
 DS_QUANTILES_SKETCH
 APPROX_QUANTILE_FIXED_BUCKETS
+# Syntex
```

Comment:
what's # Syntex?

Reply:
This is a typo. It was supposed to say "#syntax". I changed it to "#operators".